index.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const numApproved = 2;
+
+module.exports = robot => {
+	robot.on('pull_request.opened', async context => {
+	    console.log("SOMETHING IS HAPPENING");
+	    // Array of all pull requests and getting the first object in the array
+	    var usableContext = JSON.parse(JSON.stringify(context));
```

Comment:
The `context` parameter is already a JSON object,  stringifying and parsing it is redundant. do we know why we need to do this?